frontends/infinite-corridor/src/api/learning-resources/favorites.test.tsx

```diff
@@ -1,5 +1,5 @@
 import React from "react"
-import { renderHook } from "@testing-library/react-hooks/dom"
+import { renderHook, waitFor } from "@testing-library/react"
```

Comment:
Updating react required updating `@testing-library/react`; The new version of `@testing-library/react` includes `renderHook` as part of the main library, so we don't need `@testing-library/react-hooks` anymore

The `renderHook` API changed very slightly—it no longer returns `waitFor` helper.